homeassistant/components/daikin_madoka/.gitignore

```diff
@@ -0,0 +1,140 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
This file should not be here. Please remove this file.

Reply:
Done